ui/src/style/pdf-viewer.less

```diff
@@ -14,5 +14,12 @@
  * Hide default annotations to avoid clutter and confusion
  */
 .annotationLayer {
-  display: none;
+  display: inline;
 }
+
+/**
+  * Hide default annotation specifically for citations to avoid unnecessary jumps
+  */
```

Comment:
Would a more accurate comment here specify that we're hiding annotations for citations from PDF.js to make ours the default?